wsdl/ver10/recording.wsdl

```diff
@@ -683,6 +690,36 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:attribute>
 				<xs:anyAttribute processContents="lax"/>
 			</xs:complexType>
+			<xs:element name="OverrideSegmentDurationRequest">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>   <!-- first ONVIF then Vendor -->
+					</xs:sequence>
+					<xs:attribute name="TargetSegmentDuration" type="xs:duration" minOccurs="1">
+						<xs:annotation>
+							<xs:documentation>The new target duration for recorded segments.</xs:documentation>
+						</xs:annotation>
+					</xs:attribute>
+					<xs:attribute name="Expiration" type="xs:duration" minOccurs="1">
+						<xs:annotation>
+							<xs:documentation>The length of time during which the override request is to be in effect.</xs:documentation>
+						</xs:annotation>
+					</xs:attribute>
+					<xs:attribute name="RecordingConfiguration" type="tt:RecordingReference" minOccurs="1">
+						<xs:annotation>
+							<xs:documentation>The recording configuration for which the override is requested.</xs:documentation>
+						</xs:annotation>
+					</xs:attribute>
+					<xs:anyAttribute processContents="lax"/>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="OverrideSegmentDurationResponse">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>   <!-- first ONVIF then Vendor -->
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
```

Comment:
same holds for the response. Here e.g. microsoft requires to pass by reference an output parameter.